### PR TITLE
DTGB-691: Removed link to OSM and added example screenshots for the m…

### PR DESCRIPTION
**Pull request by meirege**
…aps component.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [x] I have updated the documentation accordingly.
- [x] I have updated the styleguide CHANGELOG accordingly.

### CODE REVIEW

File: components/31-molecules/map/map.twig [hunk 1]
```diff
@@ -1,16 +1,25 @@
...
+
+{% include '@heading' with {
+  'type': 'h2',
+  'heading_text': 'Map example'
+} %}
+
+{% include '@image' with {
+  src: '/styleguide/img/stock/map1.png' | path,
+  modifier: 'gentinfo-logo',
+  alt_text: 'Gent info',
```
Comment (hstandaert) [inline components/31-molecules/map/map.twig]: I suppose this is not the Gentinfo logo? 🤨

File: components/31-molecules/map/map.twig [hunk 2]
```diff
@@ -1,16 +1,25 @@
...
+
+{% include '@heading' with {
+  'type': 'h2',
+  'heading_text': 'Map example with popup'
+} %}
+
+{% include '@image' with {
+  src: '/styleguide/img/stock/map2.png' | path,
+  modifier: 'gentinfo-logo',
+  alt_text: 'Gent info',
```
Comment (hstandaert) [inline components/31-molecules/map/map.twig]: Ditto.

**Review by hstandaert — CHANGES_REQUESTED** [review]

File: components/31-molecules/map/map.twig (hunk 1 above)
Comment (meirege) [reply components/31-molecules/map/map.twig]: Fixed 😇

File: components/31-molecules/map/map.twig (hunk 2 above)
Comment (meirege) [reply components/31-molecules/map/map.twig]: Fixed 😇

**Review by hstandaert — APPROVED** [review]

**Pull request merged**